### ISSUE TITLE
Lecture 5 - Slide 5 - Definition

### ISSUE THREAD

**Issue by angelangweihan** [jakelever/textasdata_bugbounty#32]
Definition of over-fitting not included in slides

**Comment by jakelever** [comment]
Yeah, I can see a bit of confusion coming from there. There's more info in the Lecture 4 extra slides that we didn't get to. But I think you're right to add it there to. So fixed and 2 points as it's a little confusing
